spec/acceptance/mssql_spec.rb

```diff
@@ -40,6 +40,20 @@
             },
           }
         },
+        {
+          name: "auxiliary/scanner/mssql/mssql_version",
+          platforms: [:linux, :osx, :windows],
+          targets: [:session, :rhost],
+          skipped: false,
+          lines: {
+            all: {
+              required: [
+                "Version: ",
+                "Encryption is "
```

Comment:
Could we throw regexes here, similar to some of the other snippets you've got:

```suggestion
                /Version: Microsoft SQL Server \d+.\d+/,
                /Encryption: (?:on|off|or whatever)/
```